_pages/en_US/installing-boot9strap-(mset9-cli).txt

```diff
@@ -36,7 +36,7 @@ In this section, you will prepare the SD card data necessary for the MSET9 explo
 1. Open Mii Maker
 1. Wait for your console to reach the "Welcome to Mii Maker" screen, then exit Mii Maker
     + You may see [this screen](/images/screenshots/mset9/mii-extdata.png), which indicates the necessary data has been created
-    + If you just reach the normal Mii Maker screen, exit Mii Maker and continue to the next step
+    + If you just reach the normal Mii Maker screen, or you see your Miis, exit Mii Maker and continue to the next step
```

Comment:
Probably better to hyperlink Mii Maker screen to a picture of the mii maker screen

Reply:
Added two images, the welcome screen and the existing screen